setup.py

```diff
@@ -37,7 +37,7 @@
 requires = ['docutils>=0.3', 'numpy>=1.5.1', 'scipy>=0.14',
             'python-geotiepoints>=1.1.1',
             'h5py>=2.5', 'requests', 'six', 'pyyaml',
-            'appdirs']
+            'appdirs', 'git-lfs']
```

Comment:
If the git LFS is only needed for the docs then shouldn't this maybe be moved an extra_requires or be conditional on being on the RTD environment?

Reply:
Agree, fixed. Had forgot my rtd_requirerments.txt file